src/pylorax/api/compose.py

```diff
@@ -848,6 +848,28 @@ def compose_args(compose_type):
                                  "app_template":            None,
                                  "app_file":                None,
                                 },
+            "hyper-v":          {"make_iso":                False,
```

Comment:
This section needs to have all the new fields we've added, like tar_disk_name, etc.

Reply:
oops. Fixed.